remote_execution/oss/re_grpc/src/client.rs

```diff
@@ -312,6 +298,33 @@ impl REClientBuilder {
             return Err(anyhow::anyhow!("Server has remote execution disabled."));
         }
 
+        let max_decoding_msg_size = opts.max_decoding_message_size
+            .unwrap_or(capabilities.max_msg_size * 2);
```

Comment:
It seems like this would allow the user to configure a value that is actually too low and thus will lead to errors. In the case where that happens, should we either be taking the max or maybe throwing an error?

Reply:
We could assert a minimum of let's say 1MB? There is a check though to ensure that the max grpc decoding size is strictly higher than the max batch size.

Reply:
Oh, sorry, my eyes somehow glazed over that check, nevermind me